test-scripts/0.2.0/export-env-vars.fake-mesh.example.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash 
+
+export NHAIS_MESH_MAILBOX_ID=gp_mailbox
+export NHAIS_MESH_CYPHER_TO_MAILBOX="XX11=nhais_mailbox
+TES5=nhais_mailbox
+YY21=nhais_mailbox"
```

Comment:
multiple nhais_mailbox in the same variable?

Reply:
Its a valid use-case. PCRM for example will only have a single mailbox for all cypher. Current/legacy NHAIS has one mailbox per NHAIS instance.